integration-test/1972-big-parking-lots.py

```diff
@@ -0,0 +1,32 @@
+# -*- encoding: utf-8 -*-
+import dsl
+from shapely.wkt import loads as wkt_loads
+from tilequeue.tile import deg2num
+from . import FixtureTest
+
+
+def _tile_centre(z, x, y):
+    from tilequeue.tile import num2deg
+    lat, lon = num2deg(x + 0.5, y + 0.5, z)
+    return (lon, lat)
+
+
+class BigParkingLotsTest(FixtureTest):
+
+    def test_osm_big_parking_capacity(self):
+        lon, lat = (8.9140471, 48.8352716)
+        self.generate_fixtures(
+            dsl.point(8712714905, (lon, lat), {
+                u"amenity": u"parking",
+                u"capacity": u"6472217472217",
+                u"parking": u"surface",
+                u"source": u"openstreetmap.org",
+            })
+        )
+
+        x, y = deg2num(lat, lon, 15)
+        self.assert_no_matching_feature(
+            15, x, y, 'places', {
```

Comment:
A zoom 15 tile should contain all the zoom 15+ content, so should we instead assert (like in `test_meta.py` change) that it be included but with a **min_zoom** of `18`?

Reply:
Looks like most our tests are zoom 16 max zoom, so I'd update the assertion here to test 16 not 15 (and line 27 above).

Reply:
I modified the test to check for the presence of a feature with a min_zoom of 18, but it doesn't seem to pass locally because the `pois` layer is completely empty:

```
AssertionError: Did not find feature including properties {'source': u'openstreetmap.org', 'min_zoom': 18, 'id': 8712714905} (because layer 'pois' was empty)
```

Am I missing something in the updated test?